### DISCUSSION TITLE
Addition of older data to the API (pre- and post-beta)?

### DISCUSSION THREAD

**Discussion by rreaser** [ercot/api-specs#43] [General]
Hello,

Thanks for developing this! It's been awesome to test out so far.

I saw it's been mentioned that you may add data going back earlier than December 2023 during the beta testing. That would be great! I have two related questions:

1. Once the API is out of beta, will older data be added, regardless?
2. If so, is there a rough timeline of when this would happen?

Thanks again! As an energy researcher, I am always very impressed with ERCOT's transparency and data availability.

**Comment by mackermann-ercot** [comment]
Hi rreaser, 

Thank you for the kind words.  We are always looking for suggestions and feedback to improve our products.  To answer your questions: 
 
   1. Public API is now officially out of beta and there may be some EMIL Products that will be updated with earlier data sets (prior to December 2023), however those still have yet to be determined.

   2. This will depend on the demand (per product) and available resources for the efforts. There has been no official word as to when/if any of the current products will be updated with data prior to December 2023.

Regards,
mackermann, ERCOT

**Reply by rreaser** [reply]
This is super helpful info! Thanks for the reply!
